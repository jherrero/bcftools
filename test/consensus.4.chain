chain 500 1 501 + 1 501 1 504 + 1 504 1
15 0 3
485

chain 490 2 501 + 0 501 2 490 + 0 490 2
199 11 0
291

