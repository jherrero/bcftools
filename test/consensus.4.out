>1:2-501
TACCATATGTGACATAAAATAAAAAAGAACATAACCTACGTATCAACTAAAGTGGTTGTT
TGCAGAAAAGGAAGACTTAAAAAGAGTCAGTACTAACCTACATAATATATACAATGTTCA
TTAAATAATAAAATGAGCTCATCATACTTAGGTCATCATAAATATATCTGAAATTCACAA
ATATTGATCAAATGGTAAAATAGACAAGTAGATTTTAATAGGTTAAACAATTACTGATTC
TCTTGAAAGAATAAATTTAATATGAGACCTATTTCATTATAATGAACTCACAAATTAGAA
ACTTCACACTGGGGGCTGGAGAGATGGCTCAGTAGTTAAGAACACTGACTGCTCTTCTGA
AGGTCCTGAGTTCAAATCCCAGCAACCACATGGTGACTTACAACCATCTGTAATGACATC
TGATGCCCTCTGGTGTGTCTGAAGACAGCTACAGTGTACTTACATAAAATAATAAATAAA
TCTTTAAAAACAAAAAAAAAGAA
>2
GAAGATCTTTTCCTTATTAAGGATCTGAAGCTCTGTAGATTTGTATTCTATTAAACATGG
AGAGATTAGTGATTTTCCATATTCTTTAAGTCATTTTAGAGTAATGTGTTCTTAAGATAA
ATCAGAAAAACAAAAACTTGTGCTTTCCTGTTTGAAAAACAAACAGCTGTGGGGAATGGT
GTCGGGACAGCCTTTTTATAAATAATGTTGAGGCTTTGATACGTCAAAGTTATATTTCAA
ATGGAATCACTTAGACCTCGTTTCTGAGTGTCAATGGCCATATTGGGGATTTGCTGCTGC
CAATGACAGCACACCCTGGGAATGCCCCAACTACTTACTACAAAGCAGTGTTACATGGAG
AAGATCTTCAAGAGTCTTTTTGCTAGATCTTTCCTTGGCTTTTGATGTGACTCCTCTCAA
TAAAATCCACAGTAATATAGTGAGTGGTCTCCTGCTCCAAACCAGTATTTCAGACACAGT
TAATCCAGAC
